.jenkins/infrastructure/docker/dockerfiles/linux/base/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+#
+# Please use the associated build.sh to build this Dockerfile
+
+ARG UBUNTU_CODENAME
+FROM ubuntu:${UBUNTU_CODENAME}
+
+ARG OPENENCLAVE_VERSION
+ARG UBUNTU_VERSION
+ARG UBUNTU_CODENAME
+
+# Set up out-of-proc attestation
+ENV SGX_AESM_ADDR=1
+
+# Disable interactions
+ENV DEBIAN_FRONTEND noninteractive
+
+# Copy apt preferences to pin to a specific Intel SGX version
+COPY --chmod=644 apt_preference_files/intel-sgx.pref /etc/apt/preferences.d/intel-sgx.pref
+
+# Add keyrings for required apt repositories
+COPY --chmod=644 microsoft.asc.gpg /usr/share/keyrings/msprod-keyring.gpg
+COPY --chmod=644 intel-sgx-deb.key.gpg /usr/share/keyrings/intel-sgx-keyring.gpg
+
+# Update certs
+RUN apt-get update && \
+    apt-get install -y ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
+
+# Register required apt repositories
+RUN echo "deb [arch=amd64 signed-by=/usr/share/keyrings/intel-sgx-keyring.gpg] https://download.01.org/intel-sgx/sgx_repo/ubuntu ${UBUNTU_CODENAME} main" | tee /etc/apt/sources.list.d/intel-sgx.list
+RUN echo "deb [arch=amd64 signed-by=/usr/share/keyrings/msprod-keyring.gpg] https://packages.microsoft.com/ubuntu/${UBUNTU_VERSION}/prod ${UBUNTU_CODENAME} main" | tee /etc/apt/sources.list.d/msprod.list
+
+# Install Intel SGX and Open Enclave
+RUN apt-get update && \
+    apt-get install -y \
+      libssl-dev \
```

Comment:
Can you double-check if the dev package is needed to run OE? Typically, dev packages are only needed to build applications.

Reply:
Yes you are correct, it is needed to build, but not needed to run. I'll remove this. Thanks!